### DISCUSSION TITLE
[FEAT] Add mpv detection for media

### DISCUSSION THREAD

**Discussion by michirakara** [fastfetch-cli/fastfetch#1363] [Q&A] [answered]
## Wanted features:

Detect mpv for media

## Motivation:

mpv is a very popular media player as it has 28.3k stars in github, while vlc, which is supported in fastfetch, has 14.1k stars, so it would be nice to detect mpv as a media source.
(I tried to make PR but I couldn't figure out how the DBus library is working. If someone could teach me about it, I would try to implement this.)

**Comment by CarterLi** [comment, marked as answer]
It's because mpv lacks built-in support of [MPRIS standard](https://wiki.archlinux.org/title/MPRIS)

You may enable it with some 3rd-party plugins. Try [mpv-mpris](https://archlinux.org/packages/extra/x86_64/mpv-mpris/)

**Reply by michirakara** [reply]
It worked. Thanks!
